cmd/test/test.go

```diff
@@ -180,15 +194,32 @@ func newTestCommand() *cobra.Command {
 						return fmt.Errorf("running test: %w", resultsErr)
 					}
 
+					var outputFile *os.File
+					if outputFilePath != "" {
+						outputFile, err = os.Create(outputFilePath)
+						if err != nil {
+							return fmt.Errorf("creating output file: %w", err)
+						}
+						defer outputFile.Close()
+					}
+
 					outputter := output.Get(runner.Output, output.Options{
 						NoColor:            runner.NoColor,
 						SuppressExceptions: runner.SuppressExceptions,
 						Tracing:            runner.Trace,
 						JUnitHideMessage:   viper.GetBool("junit-hide-message"),
+						File:               outputFile,
 					})
 					if err := outputter.Output(results); err != nil {
 						return fmt.Errorf("output results: %w", err)
 					}
+
+					if outputFilePath != "" {
```

Comment:
Consider instead check if `outputFile` is `nil` since that's the requirement for the code block below.

Reply:
Actually... I don't think you need this block at all. Shouldn't the code above handle if `ouput.Options.File` is `nil`?

Reply:
@lcarva I put this block in to copy the output we wrote to the file to `stdout` as well. That was part of the acceptance criteria for the linked issue: that it would be possible to output to a file and `stdout` at the same time.

I'm fine leaving it out if we don't need that behavior though.

Reply:
I commented out this bit of the code and it still behaved the same way. (With `-file` output went just to the file. Without it, output went just to stdout.)

Reply:
@lcarva I'm realizing we aren't even touching that part of the code when we pass `--output appstudio`, so it wouldn't matter. You need to do something like `--output json` to make Conftest handle output.

If we do that, running without the code block makes output happen on either the file or `stdout`, but not both - which was part of the acceptance criteria. However, leaving the code in appears to hang when I run it, so that's not good either...

Reply:
It hangs because of [this](https://github.com/enterprise-contract/ec-cli/pull/1244#discussion_r1431967846). Let me run a couple more tests here.